tests/fourcipp/test_fourc_input.py

```diff
@@ -24,13 +24,19 @@
 import contextlib
 import pathlib
 import subprocess
+import time
+from typing import Callable, Dict, List
```

Comment:
```suggestion
from collections.abc import Callable
```

You don't have to import Dict and List, that is an old standard